### ISSUE TITLE
Buzzer

### ISSUE THREAD

**Issue by Nutskin** [Jaycar-Electronics/Arduino-Starter-Kit#2]
The buzzer in the photo you provided does not look like the buzzer given in the Arduino starter kit, I'm finding it difficult to understand how to actually set this project up on the breadboard

**Comment by bhapps** [comment]
yeah the buzzer requires male to female leads (for the pins on the speaker), which don't come with the Duinotech Arduino Starter Kit

**Comment by bhapps** [comment]
> 
> 
> yeah the buzzer requires male to female leads (for the pins on the speaker), which don't come with the Duinotech Arduino Starter Kit

maybe you don't (could plug speaker straight in to the Breadboard, run the leads from adjacent pin hole)

**Comment by Phoenixrexx** [comment]
Bhapps, Great idea, however still didnt work for us.. shame.. son was excited to test the buzzer
